wpilibNewCommands/src/main/native/cpp/frc2/command/CommandPtr.cpp

```diff
@@ -50,6 +50,17 @@ CommandPtr CommandPtr::Repeatedly() && {
   return std::move(*this);
 }
 
+CommandPtr CommandPtr::Repeatedly(int times) && {
+  AssertValid();
+  std::shared_ptr<int> countPtr = std::make_shared<int>(0);
+  return std::move(*this)
+      .FinallyDo([countPtr = countPtr] { (*countPtr)++; })
```

Comment:
Any reason not to use `[countPtr] { ... }` instead of `[countPtr = countPtr] { ... }`? I think they're the same for this case, but I'm not 100% sure.

Reply:
committed, see 6e88f89